### PR TITLE
feat: add divshot deployments with circleci

### PR DESCRIPTION
**Pull request by asimpson**

### CODE REVIEW

**Comment by asimpson** [comment]
ping @cromwellryan or @robtarr for merge.

File: services/divshot/README.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  "clean_urls": true,
+  "error_page": "error.html"
+}
+```
+
+To kick off the first deploy, type `divshot push production` to push to divshot's production environment. `divshot push` will by default push to the Development environment.
+
+### Set up some authentication 
+
+[Authentication with Divshot](http://docs.divshot.com/guides/http-authentication) is a snap.
```
Comment (cromwellryan) [inline services/divshot/README.md]: Maybe add something about _Add the creds to 1password_ linking to https://github.com/sparkbox/how_to/tree/master/software/1password

File: services/divshot/README.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  override:
+    - node_modules/.bin/grunt dist
+
+deployment:
+  staging:
+    branch: master
+    commands:
+      - node_modules/.bin/divshot push production --token $DIVSHOT_TOKEN
+
+general:
```
Comment (cromwellryan) [inline services/divshot/README.md]: I've never actually used artifacts and they add build time.  Maybe yaml comment this as optional?

File: services/divshot/README.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+The `$DIVSHOT_TOKEN` in `circle.yml` is an environment variable from CircleCI. 
+
+This gives CircleCI the authority to push to Divshot. To begin, you must [get your token from Divshot](https://github.com/divshot/divshot-cli#authtoken) using the CLI.
+
+Once you have your token, the next step is to add it to CircleCI as an environment variable like so:
+
+![](http://files-misc.s3.amazonaws.com/env.gif)
+
+Once this token is set up the `deployment` command in our `circle.yml` file: `node_modules/.bin/divshot push production --token $DIVSHOT_TOKEN` will push to Divshot correctly.
+
```
Comment (cromwellryan) [inline services/divshot/README.md]: _You can run this locally to test things out_

File: services/divshot/README.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+test:
+  override:
+    - node_modules/.bin/grunt dist
+
+deployment:
+  staging:
+    branch: master
+    commands:
+      - node_modules/.bin/divshot push production --token $DIVSHOT_TOKEN
```
Comment (cromwellryan) [inline services/divshot/README.md]: Do we want to default this to staging?  Or say you can leave it out and it'll go to development?

Comment (asimpson) [inline services/divshot/README.md]: so this is weird, that command will push to prod, even though it's under "staging". Should I just change "staging" to "production"?

**Pull request merged**